### ISSUE TITLE
[BUG]: Python3.11 fails to initialize on windows

### ISSUE THREAD

**Issue by petersteneteg** [inviwo/inviwo#1358]
### Required prerequisites

- [X] Make sure you've read the [documentation](https://inviwo.org). Your issue may be addressed there.
- [X] Search the [issue tracker](https://github.com/inviwo/inviwo/issues) and [Discussions](https://github.com/inviwo/inviwo/discussionss) to verify that this hasn't already been reported. +1 or comment there if it has.
- [X] Consider asking first in [Slack](https://join.slack.com/t/inviwo/shared_invite/enQtNTc2Nzc2NDQwNzIxLTRiMWM1ZWJiYjljZjkwNWE3OTk3MzYxODZlMDUyMzRmZjUzMzBiZjVhNTM3NWUyNzU1MjI4OWJjMzdkODViMzM) or in a [Discussion](https://github.com/inviwo/inviwo/discussionss).

### Problem description

basically when we initialize our python module, pybind will load some dumpy api pointers, basically here
```
    static npy_api lookup() {
        module_ m = module_::import("numpy.core.multiarray");
        auto c = m.attr("_ARRAY_API");
 ```
https://github.com/pybind/pybind11/blob/bbb89da152a0e4f63a4fecc0d4db44f0ad855fdf/include/pybind11/numpy.h#L264
and "numpy.core.multiarray" fails to load for unknown reasons. 
Works in 3.10.8

### Inviwo version

fc4db79246a5cdfc54493a250c8d95974d94718a

### Operating system?

Windows

**Comment by petersteneteg** [comment]
Seems to work in later 3.11 versions...
